website/project/decorators.py

```diff
@@ -438,20 +438,6 @@ def wrapped(*args, **kwargs):
     # Return decorated function
     return wrapped
 
-def http_error_if_disk_saving_mode(func):
```

Comment:
Seems like this would be worth mentioning in `Changes` as well as the other places there's deleted code.

Reply:
This wasn't being used, I'll add it to changes.